packages/wallets/src/components/WalletActions/WalletActions.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import IcCashierAdd from '../../public/images/ic-cashier-deposit.svg';
+import IcCashierWithdrawal from '../../public/images/ic-cashier-withdrawal.svg';
+import IcCashierTransfer from '../../public/images/ic-cashier-transfer.svg';
+import IcCashierStatement from '../../public/images/ic-cashier-statement.svg';
+import { useWalletAccountsList } from '@deriv/api';
+import './WalletActions.scss';
+
+const getWalletHeaderButtons = (is_demo: boolean, handleAction?: () => void) => {
```

Comment:
@thisyahlen-deriv Let's avoid doing these kinds of implementations 🙏🏻 We can simply render things in a readable way in the JSX.

```tsx
{is_virtual && <button>Reset balance</button>}
{!is_virtual && <button>Deposit</button>}
{!is_virtual && <button>Withdraw</button>}
<button>Transfer</button>
<button>Transactions</button>
```

Reply:
This is for future use as well 👀